### ISSUE TITLE
Can't crop with enabled system navigation gestures

### ISSUE THREAD

**Issue by Ddystopia** [SimpleMobileTools/Simple-Gallery#2757]
OnePlus Nord 2, Android 12.

When I try to select the crop area, the fields are on the sides of the screen, and when I try to move them, the system navigation gesture (back) is performed instead of changing the selection area.

![Screenshot_2023-01-28-19-54-51-92_29b563cd0bc4fd07bd0c105b17b28204_1](https://user-images.githubusercontent.com/63151578/215285848-1ab619f8-dd23-4834-9f6b-da7a4f413ee2.jpg)

**Comment by nik1a** [comment]
Yes increasing the area that you can grab would probably fix that issue.
![and](https://user-images.githubusercontent.com/28010261/215339398-31edd7ab-61dc-44d9-8762-901672eacec5.jpg)

@Ddystopia a small tip. Usually the top 20-40%(depending on the version) of the screen is not used for those gestures. so you can drag there from the edge.

**Comment by xDonatello** [comment]
I have the same annoyance, I can't crop photos with Simple Gallery, I have to use Google Photo.
Me too with OnePlus Nord 2.

Adding a slight space (padding) to the right and left might resolve the problem.

**Comment by maptile** [comment]
I am experiencing the same problem with Lineage OS 20.0, where the screen edges are being inadvertently touched. I believe adding more margin to the image will resolve this issue. To illustrate, I have included screenshots of Snapseed and Simple Gallery Pro. As you can see, Snapseed has more margin around the image, which prevents accidental touches.

![screenshot-of-snapseed](https://user-images.githubusercontent.com/28569363/228461166-48c7767b-b764-4342-84db-e70e1d5af81c.jpg)

![screenshot-of-simple-gallery-pro](https://user-images.githubusercontent.com/28569363/228460921-384debfe-5cb7-49ef-b4ce-e7b5dc69dca5.png)

**Comment by Ddystopia** [comment]
> A temporary solution is to hold your finger on the edge of the screen for a moment and then drag inwards. That way the back gesture won't be triggered.

Unfortunately, it doesn't work for me
